components/form/TextField.tsx

```diff
@@ -7,22 +7,38 @@ import {
   Text,
   Input,
   InputProps,
+  InputGroup,
+  InputLeftElement,
 } from '@chakra-ui/react';
 
 type Props = InputProps & {
   readonly name: string;
-  readonly label: string;
+  readonly label: string | ReactNode;
   readonly required?: boolean;
+  readonly monetaryInput?: boolean;
 };
 
 const TextField: FC<Props> = props => {
-  const { name, label, required, children, ...inputProps } = props;
+  const { name, label, required, children, monetaryInput, ...inputProps } = props;
   const [field, meta] = useField(name);
 
   return (
     <FormControl isInvalid={!!meta.error && meta.touched} isRequired={required}>
       <FormLabel htmlFor={name}>{label}</FormLabel>
-      <Input {...field} {...inputProps} />
+      {monetaryInput ? (
+        <>
```

Comment:
I think you can remove the `<>` and `</>` wrapping the `InputGroup` and `Input` on line 39